app/components/container-subpod/template.hbs

```diff
@@ -1,4 +1,3 @@
-{{action-menu model=model showPrimary=false}}
```

Comment:
^-- I think this menu should stay, and can have regular dots without tooltips/actions below --v